src/test/java/com/epam/dataservice/PoliceForceServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.dataservice;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PoliceForceServiceTest {
+
+    private PoliceForceService service;
+
+    @Before
+    public void init(){
+        service = new PoliceForceService();
```

Comment:
you can create instance direct on declaration line. it's no reasonable to create instance every time before running each test